site/assets/js/main.js

```
@@ -93,17 +93,17 @@ function initializeScrollEffects() {
           entry.target.style.transitionDelay = `${delayCounter * 100}ms`;
         }
-        
+
         entry.target.classList.add('animate-in');
-        
+
         delayCounter++;
-        
+
         // Reset the counter when batch finishes
         clearTimeout(resetDelayTimeout);
```

ESLint

[error] 'clearTimeout' is not defined.

(no-undef)